lib/autobuild/packages/python.rb

```diff
@@ -0,0 +1,79 @@
+require 'autobuild/configurable'
+
+# Main Autobuild module
+module Autobuild
+    def self.python(opts, &proc)
+        Python.new(opts, &proc)
+    end
+
+    # Handler class to build python-based packages
+    class Python < Configurable
+        attr_accessor   :buildflags
+        attr_accessor   :installflags
+
+        @builddir = 'build'
+
+        def configurestamp
+            "#{builddir}/configure-autobuild-stamp"
+        end
+
+        def initialize(options)
+            @buildflags = []
+            @installflags = []
+            super
+        end
+
+        def prepare_for_forced_build
+            super
+            FileUtils.rm_f configurestamp
+            @forced = true
+        end
+
+        # There is nothing to configure
+        def configure
+            super {}
```

Comment:
`super <empty block>` ? Leftover ?

Reply:
The base class yields without checking block_given?.